packages/iov-core/src/multichainsigner.spec.ts

```diff
@@ -173,6 +174,80 @@ describe("MultiChainSigner", () => {
 
       signer.shutdown();
     });
+
+    it("can add two chains and send on both of them", async () => {
+      pendingWithoutBnsd();
+      pendingWithoutEthereum();
+
+      const profile = new UserProfile();
+      const signer = new MultiChainSigner(profile);
+      expect(signer.chainIds().length).toEqual(0);
+
+      await signer.addChain(bnsConnector(bnsdTendermintUrl));
+      await signer.addChain(ethereumConnector(httpEthereumUrl, undefined));
+      const [bnsId, ethereumChainId] = signer.chainIds();
+
+      // Create sender identities
+      const { mainWalletId, faucet: bnsFaucet } = await addWalletWithFaucet(profile, bnsId);
+      const secpWallet = profile.addWallet(
+        Secp256k1HdWallet.fromMnemonic(
+          "oxygen fall sure lava energy veteran enroll frown question detail include maximum",
+        ),
+      );
+      const ganacheMainIdentity = await profile.createIdentity(
+        secpWallet.id,
+        ethereumChainId,
+        HdPaths.ethereum(0),
+      );
+
+      {
+        // Send on BNS
+        const sendOnBns: SendTransaction = {
+          kind: "bcp/send",
+          creator: bnsFaucet,
+          recipient: await randomBnsAddress(),
+          memo: `MultiChainSigner style (${Math.random()})`,
```

Comment:
Is there a good reason to use a random value in this test?

Reply:
No. It was copied from the test above where we use a random memo to ensure we find the same transaction afterwards using searchTx. But in this case we don't do that.